h2/config.py

```diff
@@ -5,6 +5,7 @@
 
 Objects for controlling the configuration of the HTTP/2 stack.
 """
+import logging
```

Comment:
No need for the logging import if `DummyLogger` no longer subclasses from `logging.Logger`.